lib/hocon/impl/parseable.rb

```diff
@@ -417,13 +430,13 @@ def relative_to(filename)
         sibling = File.new(filename)
       else
         # this may return nil
-        sibling = Parseable.relative_to(@input, filename)
+        sibling = Hocon::Impl::Parseable.relative_to(@input, filename)
```

Comment:
Nevermind, just realized this is in the `ParseableFile` class and `relative_to` is in the Parseable module